classes/PodsInit.php

```diff
@@ -511,27 +515,61 @@ public function register_assets() {
 
 		if ( ! wp_script_is( 'jquery-ui-timepicker', 'registered' ) ) {
 			wp_register_script(
-				'jquery-ui-timepicker', PODS_URL . 'ui/js/timepicker/jquery-ui-timepicker-addon.min.js', array(
+				'jquery-ui-timepicker',
+				PODS_URL . 'ui/js/timepicker/jquery-ui-timepicker-addon.min.js', array(
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 42f507a98dab971c7de21864497083aa</sub>